src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,55 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JsonParserTest {
+    private Parser parser;
+    Cart nastyaCart = new Cart("nastya-cart");
+    RealItem car = new RealItem();
+    File file;
+
+    @BeforeEach
+    public void setUp() {
+        parser = new JsonParser();
+
+    }
+
+    @Test
+    void writeToFile() {
+        file = new File("src/main/resources/nastya-cart.json");
+
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
+        nastyaCart.addRealItem(car);
+        nastyaCart.showItems();
+
+        parser.writeToFile(nastyaCart);
+
+        assertTrue(file.exists());
```

Comment:
Always add second parameter in assertTrue/assertFalse, otherwise if test fails, you will get message like: Expected true, but found false. As second parameter you can add readable message and it will be clear what actually is broken